### DISCUSSION TITLE
Uploaded byte display for uploading to the Workshop

### DISCUSSION THREAD

**Discussion by RBMKBlaze** [SmartlyDressedGames/Unturned-3.x-Community#4632] [Ideas]
A small but useless information, since when you upload something to the workshop there is no progress message and people can get confused if their work is being uploaded, so a message like `49% (5020/10240 bytes) uploaded` would be nice to have.

**Comment by Jdance-Media** [comment]
Or at the very least something that gives the illusion of progress
